tests/test_attribute_factory_routes.py

```diff
@@ -26,19 +26,19 @@ def test_get_instances_endpoint_with_vocab_lookup(
                 {
                     "termURL": {
                         "type": "uri",
-                        "value": "https://www.cognitiveatlas.org/task/id/tsk_U9gDp8utahAfO",
+                        "value": "http://purl.bioontology.org/ontology/SNOMEDCT/1284852002",
                     }
                 },
                 {
                     "termURL": {
                         "type": "uri",
-                        "value": "https://www.cognitiveatlas.org/task/id/not_found_id",
+                        "value": "http://purl.bioontology.org/ontology/SNOMEDCT/not_found_id",
                     }
                 },
                 {
                     "termURL": {
                         "type": "uri",
-                        "value": "https://www.notanatlas.org/task/id/tsk_alz5hjlUXp4WY",
+                        "value": "http://notpurl.bioontology.org/ontology/SNOMEDCT/123456789",
```

Comment:
for clarity, how about:
```suggestion
                        "value": "http://unknownvocab.org/123456789",
```